### PR TITLE
added file parser and tests

### PR DESCRIPTION
**Pull request by Martynice**

### CODE REVIEW

File: src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> read(String path) {
+        List<String> result = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(path))) {
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java]: br - is not informative naming

File: src/main/resources/file.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+Parse file
```
Comment (RomanDubovskyi) [inline src/main/resources/file.csv]: add empty line in the end of the file

File: src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java (hunk 1 above)
Comment (Martynice) [reply src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java]: It is only in try block. Anyway, I changed

File: src/main/resources/file.csv (hunk 2 above)
Comment (Martynice) [reply src/main/resources/file.csv]: added

File: src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java]: I know, and it looks ok for now, but your methods have tendency to grow, so it's better to write correct naming from the beginning, not to refactore code in future

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**